python_ta/checkers/function_parameter_not_mentioned_checker.py

```diff
@@ -0,0 +1,59 @@
+"""checker that every function parameter is mentioned by name in the docstring text.
+"""
+
+from __future__ import annotations
+
+from typing import Optional
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+from pylint.lint import PyLinter
+
+
+class FunctionParameterNotMentionedChecker(BaseChecker):
+    """
+    A class checker to check if every function parameter is mentioned by name within it's the docstring.
+    By default, this checker will be disabled.
+    """
+
+    name = "function_parameters_not_mentioned"
+    msgs = {
+        "C9960": (
+            "The parameter '%s' is not mentioned in the docstring",
+            "function-parameters-not-mentioned",
```

Comment:
Overall, please rename to `unmentioned-parameter`. This will have changes throughout.

(This fits the naming convention of most pylint errors, which seem to be of the form <adjective> <noun>. Note that "parameter" implies "function", so the latter word is unnecessary in the name.)